install_config/topics/sssd_for_ldap_failover_configure_sssd.adoc

```diff
@@ -57,35 +57,13 @@ ldap_user_extra_attrs = mail <1>
 solution. For IPA, specify `mail`. Other LDAP solutions might use another
 attribute, such as `email`.
 
-. Modify the *_domain_* parameter in the *_/etc/sssd/sssd.conf_* file to contain
-only one domain name.
+. Confirm that the *_domain_* parameter in the *_/etc/sssd/sssd.conf_* file
+contains only one domain name.
```

Comment:
If you think it would be helpful, something like

`... contains only one domain name and a single macthing section [domain/yourdomainname.com]`

If you do not think it would be helpful, the current wording is fine.